app/controllers/job_messages_controller.rb

```diff
@@ -0,0 +1,11 @@
+class JobMessagesController < ApplicationController
+  before_filter  :authorize_only_for_admin
+
+  def show
+    @job_message = Rails.cache.fetch(params[:job_id])
```

Comment:
I wonder if there is a race condition here - the first time I tried out the button, I got an error because `@job_message` was nil. It seem this relies on the `before_enqueue` block in the job to execute first?